eslint-config-internal/index.js

```diff
@@ -37,7 +37,7 @@ module.exports = {
     "no-useless-constructor": "off",
     "global-require": "off",
     "import/no-dynamic-require": "off",
-    "import/prefer-default-export": "off",
+    'import/no-default-export': 'error',
     "import/extensions": "off",
     "max-classes-per-file": "off",
```

Comment:
do we care for consistent double-quotes `"`?)

Reply:
changed back